latch/ldata/transfer/node.py

```diff
@@ -0,0 +1,151 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Dict, List, TypedDict
+
+import graphql.language as l
+from latch_sdk_gql.execute import execute
+from latch_sdk_gql.utils import _name_node, _parse_selection
+from typing_extensions import TypeAlias
+
+from latch_cli.utils.path import get_path_error, normalize_path
+
+AccId: TypeAlias = int
```

Comment:
```suggestion
_AccId: TypeAlias = int
```

Reply:
just made the entire module private instead